tests/run_tests.sh

```diff
@@ -19,8 +19,8 @@ python_bin=$(command -v ${PYTHON_BIN})
 
 # check if python version meets our requirements
 IFS=" " read -r -a python_version <<< "$(${python_bin} -c 'import sys; print(sys.version_info[:])' | tr -d '(),')"
-if [ ${python_version[0]} -lt 3 ] || [[ "${python_version[0]}" -eq 3 && "${python_version[1]}" -lt 6 ]]; then
```

Comment:
`typing` module requires >= Python 3.5 or higher.
So, we should keep this limitation.
`typing` module is used for the completion of `list` , `dict` and class inheritance.

Reply:
Ok, I can add it back. I tested it and it seemed to work with older versions too. Maybe I did something wrong!